package.nls.json

```diff
@@ -101,6 +101,11 @@
     "AWS.running.command": "Running command: {0}",
     "AWS.samcli.detect.settings.updated": "Settings updated.",
     "AWS.samcli.detect.settings.not.updated": "No settings changes necessary.",
+    "AWS.samcli.deploy.workflow.init": "Building SAM Application...",
+    "AWS.samcli.deploy.workflow.packaging": "Packaging SAM Application to S3 Bucket: {0} with profile: {1}",
+    "AWS.samcli.deploy.workflow.stackName.initiated": "Deploying SAM Application to CloudFormation Stack: {0}",
```

Comment:
Out of curiosity, why do the other messages contain the profile, but not this message?

Reply:
Honestly, because I was using a critical eye around the calls that implement the profile, then decided to move the rest of the strings to nls. I can add this.

Reply:
✔️